src/reanimated2/jestUtils.ts

```diff
@@ -5,7 +5,10 @@ declare global {
   namespace jest {
     interface Matchers<R> {
       toHaveAnimatedStyle(
-        style: Record<string, unknown>[] | Record<string, unknown>
+        style: Record<string, unknown>[] | Record<string, unknown>,
+        config?: {
+          shouldMatchAllProps?: boolean;
```

Comment:
this looks like a breaking change for library users, we'll need to discuss it with the team